sang19.java

```diff
@@ -0,0 +1,15 @@
+//File Handling in Java
+import java.io.* ;
+import java.util.Scanner;
+import org.apache.commons.io.FileUtils; 
```

Comment:
how to install this package

Reply:
I need not install this package. Firefox displays an error message as it does not find the permissions attribute in the html file. I ignored this, and the applet began to run.